Raspi/Test/mock_wireless_module.py

```diff
@@ -0,0 +1,170 @@
+parser.add_argument('-r', '--rate', action='store', type=float, default=1, help="Rate of data in data sent per second. Default is 1 data pack sent per second.")
+parser.add_argument('--host', action='store', type=str, default="192.168.100.100", help="Address of the MQTT broker. If nothing is selected it will default to 192.168.100.100.")
+parser.add_argument('--id', action='store', type=int, default=None, help="Specify the sensor to produce fake data. eg. --id 1 specifies that sensor 1 only produces data. If nothing is given all sensors will be active." )
+
+def send_fake_data(client, duration, rate, sensorID):
+    """ Send artificial data over MQTT for each sensor chanel. Sends [rate] per second for [duration] seconds """
+
+    start_time = round(time.time(), 2)
+
+    while True:
+        current_time = round(time.time(), 2)
+        total_time = round(current_time - start_time, 2)
+
+        print() # Newline for clarity
+
+        # Wireless Sensor 1 (Middle)
+        if sensorID == 1 or sensorID == None:
+
+            # generate JSON data and publish it over MQTT
+            data = {
+                "CO2": Sensors.CO2(),
+                "temperature": Sensors.temperature(),
+                "humidity": Sensors.humidity(),
+                "accelerometer": Sensors.accelerometer(),
+                "gyroscope": Sensors.gyroscope(),
+                "battery": Sensors.battery(),
+                "time": Sensors.time()}
+
+            JSON_data = json.dumps(data)
+            pub_path = "v3/wireless-sensor/1/data"
+
+            client.publish(pub_path, JSON_data)
+            print(pub_path, "--> ", JSON_data)
+
+        # Wireless Sensor 2 (client, Back)
+        if sensorID == 2 or sensorID == None:
+
+            # generate JSON data and publish it over MQTT
+            data = {
+                "CO2": Sensors.CO2(),
+                "reedSwitch": Sensors.reedSwitch(),
+                "GPS": Sensors.GPS(),
+                "temperature": Sensors.temperature(),
+                "battery": Sensors.battery(),
+                "time": Sensors.time()}
+
+            JSON_data = json.dumps(data)
+            pub_path = "v3/wireless-sensor/2/data"
+
+            client.publish(pub_path, JSON_data)
+            print(pub_path, "--> ", JSON_data)
+
+        # Wireless Sensor 3 (client, Front)
+        if sensorID == 3 or sensorID == None:
+
+            # generate JSON data and publish it over MQTT
+            data = {
+                "steering": Sensors.steering(),
+                "temperature": Sensors.temperature(),
+                "humidity": Sensors.humidity(),
+                "battery": Sensors.battery(),
+                "time": Sensors.time()}
+
+            JSON_data = json.dumps(data)
+            pub_path = "v3/wireless-sensor/3/data"
+
+            client.publish(pub_path, JSON_data)
```

Comment:
I think you can bring these two lines out from the if statement as they're just repeated!